contracts/Distributor.sol

```diff
@@ -0,0 +1,148 @@
+pragma solidity 0.5.7;
+
+import * as ERC721 from "@openzeppelin/contracts/token/ERC721/ERC721Full.sol";
+import * as IERC20 from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import * as Ownable from "@openzeppelin/contracts/ownership/Ownable.sol";
+import * as SafeMath from "./external/openzeppelin-solidity/math/SafeMath.sol";
+import * as INXMMaster from "./INXMMaster.sol";
+import * as Pool1 from "./Pool1.sol";
+import * as PoolData from "./PoolData.sol";
+import * as Claims from "./Claims.sol";
+import * as NXMToken from "./NXMToken.sol";
+import * as QuotationData from "./QuotationData.sol";
+
+contract Distributor is ERC721.ERC721Full("NXMDistributorNFT", "NXMDNFT"), Ownable.Ownable {
+
+  struct TokenData {
+    uint expirationTimestamp;
+    address lastOwner;
+    bytes4 coverCurrency;
+    uint[] coverDetails;
+    uint coverId;
+  }
+
+  uint public constant CLAIM_VALIDITY_MAX_DAYS_OVER_COVER_PERIOD = 30 days;
+  uint public constant CLAIM_DEPOSIT_PERCENTAGE = 5;
```

Comment:
this is tokensRetained / 2: https://github.com/NexusMutual/smart-contracts/blob/master/contracts/QuotationData.sol#L67

Reply:
given it's hard coded in the constructor in QuodationData, for now we don't have a choice, so we can keep it as is